.mia-template/README.md

```diff
@@ -0,0 +1,17 @@
+# mia_template_service_name_placeholder
+
+This is a simple C++ application template.
+
+## Development Local
+
+To start the application locally
+
+```bash
+  mkdir build && cd build
+  cmake ..
+  make
+  ./mia-platform-cpp-helloworld-exe  
+
+```
+
+By default the service will run on port 3000, to change the port please set `HTTP_PORT` env variable
```

Comment:
This is not taken from environemnt variables, meaning that this statement is false, isn't it?